hack/tools.go

```diff
@@ -16,4 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package test
+package hack
```

Comment:
why does git think this file was renamed from `verify/boilerplate/test/fail.go`?